sample/Sample/ContentPopup/ContentPopupBGColorTest.xaml

```diff
@@ -17,6 +18,7 @@
                     HeightRequest="200"
                     HorizontalOptions="Center"
                     Text="Show Popup"
+                    TextColor="White"
```

Comment:
Are those TextColor and BackgroundColor not changeable?

Reply:
As the backend UI changed from Elmsharp to NUI, the default colors of the platform has been changed, affecting the test case so that some controls were not visible.
So I've fixed some colors can be affect on test.